js/app/DialogueEvaluation.js

```diff
@@ -2,7 +2,8 @@
 
 var $ = require( 'jquery' ),
 	LicenceStore = require( './LicenceStore' ),
-	licences = new LicenceStore( require( './LICENCES' ) ),
+	config = require( '../config.json' ),
+	licences = new LicenceStore( require( './LICENCES' ), config.portedLicences ),
```

Comment:
I started to think that maybe we should have put those ported license URLs to a separate file, not into the config file but this can be easily changed later, so I wouldn't bother with moving it now.